packages/st2mistral/debian/mistral-api.init

```diff
@@ -20,7 +20,8 @@ DESC="mistral"
 NAME=mistral-api
 DAEMON=/opt/stackstorm/mistral/bin/gunicorn
 COMPONENTS="api,engine,executor"
-PIDFILE=/var/run/mistral/$NAME.pid
+RUN_DIR="/run/mistral"
```

Comment:
Sure, this part of the change has been reverted and all PIDFILES are back in /var/run/.